### PR TITLE
Tutorial/candle

### PR DESCRIPTION
**Pull request by mturilli**
First iteration on the documentation.

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -5,31 +5,110 @@
...
+* virtualenv >= 16
+* pip >= 19
 
 or
-* Anaconda Python 2.7
 
-If you plan to use RADICAL-Pilot on remote machines, you would also require to
-setup a password-less ssh login to the particular machine. 
-(`help <http://www.debian-administration.org/article/152/Password-less_logins_with_OpenSSH>`_)
+* Anaconda Python >= 3.5
```
Comment (andre-merzky) [inline docs/source/installation.rst]: That should be 2.7, too?

File: docs/source/installation.rst [hunk 2]
```diff
@@ -38,15 +117,15 @@ RADICAL-Pilot is distributed via PyPi and Conda-Forge. To install RADICAL-Pilot
...
     virtualenv --system-site-packages $HOME/ve
     source $HOME/ve/bin/activate
     pip install radical.pilot
 
 via Conda-Forge
------------------
+---------------
 .. code-block:: bash
 
     conda create -n ve -y python=2.7
```
Comment (andre-merzky) [inline docs/source/installation.rst]: I think the `source activate ve` line changed, but @iparask should confirm / refute that.

File: docs/source/profiles.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        pdesc = rp.ComputePilotDescription()
+        pdesc.resource = "local.localhost"
+        pdesc.runtime  = 5 # minutes
+        pdesc.cores    = 8
+        pdesc.cleanup  = False
+        pdesc._config  = {'number_of_workers' : {'StageinWorker'   :  1,
+                                                 'ExecWorker'      :  2,
+                                                 'StageoutWorker'  :  1,
+                                                 'UpdateWorker'    :  1},
+                          'blowup_factor'     : {'Agent'           :  1,
```
Comment (andre-merzky) [inline docs/source/profiles.rst]: We probably should remove documentation for `pdesc._config` - that should not be exposed to the end user at the moment.

File: docs/source/installation.rst (hunk 1 above)
Comment (mturilli) [reply docs/source/installation.rst]: Done. Thanks.

File: docs/source/profiles.rst (hunk 3 above)
Comment (mturilli) [reply docs/source/profiles.rst]: Done. I muted the whole section from the index.

File: docs/source/installation.rst (hunk 2 above)
Comment (mturilli) [reply docs/source/installation.rst]: @iparask confirmed that is the right line.

Comment (andre-merzky) [reply docs/source/installation.rst]: Thanks for checking!

File: docs/source/profiles.rst (hunk 3 above)
Comment (andre-merzky) [reply docs/source/profiles.rst]: Makes sense, thanks.

**Comment by andre-merzky** [comment]
This PR is now targeting `py2/devel` (branched off the last py2 release), and can be merged IMHO.

**Pull request merged**